fastlane/Pluginfile

```diff
@@ -3,3 +3,4 @@
 # Ensure this file is checked in to source control!
 
 gem 'fastlane-plugin-git_status'
+gem 'fastlane-plugin-firebase_app_distribution'
```

Comment:
After adding the plugin to Fastlane, I saw this message:

```
[✔] 🚀 
[14:50:37]: It seems like you wanted to load some plugins, however they couldn't be loaded
[14:50:37]: Please follow the troubleshooting guide: https://docs.fastlane.tools/plugins/plugins-troubleshooting/
[14:50:38]: Plugin 'fastlane-plugin-firebase_app_distribution' was added to './fastlane/Pluginfile'
[14:50:38]: It looks like fastlane plugins are not yet set up for this project.
[14:50:38]: fastlane will modify your existing Gemfile at path '/Users/yramocan/Development/fastlane-template/Gemfile'
[14:50:38]: This change is necessary for fastlane plugins to work
[14:50:38]: Should fastlane modify the Gemfile at path '/Users/yramocan/Development/fastlane-template/Gemfile' for
you? (y/n)
n
[14:52:11]: Please add the following code to '/Users/yramocan/Development/fastlane-template/Gemfile':

plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
eval_gemfile(plugins_path) if File.exist?(plugins_path)

[!] Please update '/Users/yramocan/Development/fastlane-template/Gemfile and run fastlane again
```

I noticed that the `Gemfile` does not have this already, as suggested by this message. Do we need to modify the `Gemfile` with the suggested changes?

Reply:
It means that it needs to be in the Gemfile for every team using the template. We can make sure it's in the template Gemfile as well.